blocks/table/table.css

```diff
@@ -0,0 +1,134 @@
+/* stylelint-disable no-descending-specificity */
+.table-container {
+    margin-bottom: 32px;
+}
+
+.table-container > div.table-wrapper {
+    margin-bottom: 0;
+}
+
+.table-container > .table-wrapper + .table-wrapper {
+    margin-top: 0;
+}
+
+.table-form-submit-wrapper {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+.table-form-submit-wrapper button {
+    height: 50px;
+    width: 300px;
+    background-color: var(--blue);
+    color: var(--white);
+    border-radius: 6px;
+    font-family: var(--heading-font-family);
+    font-size: var(--heading-size-s);
+    margin: 20px 0;
+}
+
+.table-form-submit-wrapper button:disabled {
+    background-color: var(--blue-light);
+}
+
+.table-wrapper:not(:has(+ .table-wrapper)) .table tbody tr:last-of-type td {
+    border-bottom: 2px solid var(--blue);
+}
+
+.table {
+    width: 100vw;
+    margin-left: calc(-50vw + 50%);
+}
+
+.table:first-of-type {
+    border-top: 2px solid var(--blue);
+}
+
+.table table {
+    border-spacing: 0;
+    width: 100%;
+}
+
+.table thead th {
+    padding: 0 12px;
+    border-bottom: 2px solid var(--blue);
+    font-family: var(--heading-font-family);
+    font-size: var(--heading-size-xs);
+    text-transform: uppercase;
+    background-color: var(--blue-lightest);
+}
+
+.table thead th:first-child {
+    text-align: left;
+    border-right: 2px solid var(--blue);
+}
+
+.table thead th:last-child {
+    width: 125px;
+}
+
+.table tbody tr td {
+    padding: 16px 12px;
+    border-bottom: 2px solid var(--blue);
+}
+
+.table tbody tr:last-child td {
+    border-bottom: none;
+}
+
+.table tbody tr td:first-child {
+    border-right: 2px solid var(--blue);
+    position: relative;
+}
+
+.table table tbody tr td:first-child p:not([class]) {
+    margin-right: 30px;
+    margin-bottom: 0;
+    font-size: var(--body-size-s);
+}
+
+.table tbody tr td:first-child .img-wrapper {
+    position: absolute;
+    display: flex;
+    flex-flow: column;
+    justify-content: center;
+    align-items: center;
+    width: 40px;
+    top: 50%;
+    right: 0;
+    bottom: 50%;
+}
```

Comment:
historically, this information has been with the title of the item. is move an intentional choice?

Reply:
Yes, I moved the dietary icons to the right edge of the first column instead of the icons rendering in the title. 
![Screen Shot 2024-10-01 at 11 27 16 AM](https://github.com/user-attachments/assets/1f7611d4-1271-45eb-a685-d463a0fd6d80)